### PR TITLE
Changed reapeating Mail to Url Management in 11.1.1 General, Control panels list

### PR DESCRIPTION
**Pull request by curiousyuvi**
In page:
[Mastering Plone 6 Development > 11. Configuring and Customizing Plone "Through The Web" > 11.1. The Control Panel > 11.1.1. General](https://training.plone.org/mastering-plone/configuring_customizing.html#general)

In the control panels list, changed the repeating **Mail** to **Url Management**

### CODE REVIEW

File: docs/mastering-plone/configuring_customizing.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Other control panels, e.g. Content Rules still need to be implemented.
 1. Date and Time
 1. Language
 1. Mail
-1. Mail
+1. Url Management
```
Comment (stevepiercy) [inline docs/mastering-plone/configuring_customizing.md:54]: URL is an acronym.

```suggestion
1. URL Management
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/mastering-plone/configuring_customizing.md (hunk 1 above)
Comment (curiousyuvi) [reply docs/mastering-plone/configuring_customizing.md:54]: Ok, I'll make those changes.

**Review by stevepiercy — APPROVED** [review]
Thank you! Please go ahead and merge.

**Comment by curiousyuvi** [comment]
@stevepiercy  I don't have merging rights, can you please merge this.

**Pull request merged**